src/smepmp.adoc

```diff
@@ -77,95 +40,88 @@ A _Shared-Region_ rule is *enforced* on all modes, with restrictions depending o
 |{set:cellbgcolor:!} 0|0|0|0 2+|Inaccessible region (Access Exception)
 |0|0|0|1|Access Exception|Execute-only region
 |0|0|1|0 2+|Shared data region: Read/write on M mode, read-only on S/U mode
-|0|0|1|1 2+|Shared data region: Read/write for both M and S/U mode
+|0|0|1|1 2+|Shared data region: Read/write for both M and S/U modes
 |0|1|0|0|Access Exception|Read-only region
 |0|1|0|1|Access Exception|Read/Execute region
 |0|1|1|0|Access Exception|Read/Write region
 |0|1|1|1|Access Exception|Read/Write/Execute region
-|1|0|0|0 2+|Locked inaccessible region* (Access Exception)
-|1|0|0|1|Locked Execute-only region*|Access Exception
-|1|0|1|0 2+|Locked Shared code region: Execute only on both M and S/U mode.*
-|1|0|1|1 2+|Locked Shared code region: Execute only on S/U mode, read/execute on M mode.*
-|1|1|0|0|Locked Read-only region*|Access Exception
-|1|1|0|1|Locked Read/Execute region*|Access Exception
-|1|1|1|0|Locked Read/Write region*|Access Exception
-|1|1|1|1 2+|Locked Shared data region: Read only on both M and S/U mode.*
+|1|0|0|0 2+|*Locked* inaccessible region (Access Exception)
+|1|0|0|1|*Locked* Execute-only region|Access Exception
+|1|0|1|0 2+|*Locked* Shared code region: Execute only on both M and S/U mode.
+|1|0|1|1 2+|*Locked* Shared code region: Execute only on S/U mode, read/execute on M mode.
+|1|1|0|0|*Locked* Read-only region|Access Exception
+|1|1|0|1|*Locked* Read/Execute region|Access Exception
+|1|1|1|0|*Locked* Read/Write region|Access Exception
+|1|1|1|1 2+|*Locked* Shared data region: Read only on both M and S/U mode.
 |===
 
-*: *Locked* rules cannot be removed or modified until a *PMP reset*, unless ``mseccfg.RLB`` is set.
-
-==== Visual representation of the proposal
-
-image::smepmp-visual-representation.png[]
+[TIP]
+====
+Shared data regions may be used to expose static platform-specific information (e.g. ACPI tables or the device tree), used by both Machine and Supervisor modes, or exchange data between them without the overhead of copying them.
 
-=== Smepmp software discovery
+Shared code regions may be used for exposing vendor-specific functionality (e.g. with custom instructions) to Supervisor/User modes, without exposing that code, or risking the posibility of modifications (no write access).
+====
 
-Since all fields defined on ``mseccfg`` as part of this proposal are locked when set (``MMWP``/``MML``) or locked when cleared (``RLB``), software can't poll them for determining the presence of Smepmp. It is expected that BootROM will set ``mseccfg.MMWP`` and/or ``mseccfg.MML`` during early boot, before jumping to the firmware, so that the firmware will be able to determine the presence of Smepmp by reading ``mseccfg`` and checking the state of ``mseccfg.MMWP`` and ``mseccfg.MML``.
+Additionally, when `mseccfg.MML=1`:
 
-[[rationale]]
-=== Rationale
```

Comment:
Feels like a shame to lose all this information too.

Reply:
I think the important stuff is still there, some as part of the implementation considerations, some as tips/info, and the most important part regarding rlb is mentioned twice. It seems inconsistent with the rest of the document to keep this, it's a bit too much to e.g. explain why we didn't say anything about mprv, or what was our thought process etc. The document is still there for people to dig deeper, it doesn't have to be included in the privilege spec. The chapter was 7 pages long, it's now 4 pages long and still contains what's needed for someone to implement this.